helita/sim/rh15d.py

```diff
@@ -12,33 +12,53 @@
 
 
 class Rh15dout:
-    """
-    Class to load and manipulate output from RH 1.5D.
-    """
-    def __init__(self, fdir='.', verbose=True, autoread=True):
+    def __init__(self, fdir='.', verbose=True):
```

Comment:
You are adding an old version on top of the new version. And with conflict strings... Please don't add this file, just keep the version from ITA-solar/helita master.